helpers/user-helpers.js

```diff
@@ -996,334 +1111,350 @@ module.exports = {
-                      "products.item": ObjectId(cartData.product),
-                    },
-    
-                    {
-                      $inc: { "products.$.quantity": cartData.count },
-                    }
-                    ).then((data)=>{
-    
-                        // console.log(data);
-    
-                        resolve({status:true});
-    
-                    }).catch((err)=>{
-    
-                        if(err){
+                    db.get().collection(collections.CART_COLLECTION)
+                    .updateOne(
 
-                            console.log(err);
-    
-                            reject(err);
-                            
-                        }
-    
-                    }
-                );
+                        { _id: ObjectId(cartData.cart), "products.item": ObjectId(cartData.product) },
+        
+                        { $inc: { "products.$.quantity": cartData.count } }
 
-            }
+                        ).then((data)=>{
+        
+                            resolve({status:true});
+        
+                        }).catch((error)=>{
+        
+                            console.error("Error-2 from Db update query at changeCartProductQuantity user-helpers: ", error);
 
-        })
+                            reject(error);
+        
+                        }
 
-    },
-    deleteProductFromCart:(productDetails)=>{
+                    );
 
-        return new Promise((resolve,reject)=>{
+                }
+
+            }catch(error){
+            
+                console.error("Error from changeCartProductQuantity user-helpers: ", error);
+            
+                reject(error);
+            
+            }
+
+        })
+
+    },
+    deleteProductFromCart : (productDetails)=>{
+
+        return new Promise((resolve,reject)=>{
+
+            try{
+
+                db.get().collection(collections.CART_COLLECTION)
+                .updateOne(
+
+                    { _id: ObjectId(productDetails.cart) },
+                    
+                    // Remove the product from user Cart
+                    { $pull: { products:{item:ObjectId(productDetails.product)} } }
 
-            db.get().collection(collections.CART_COLLECTION)
-            .updateOne(
-                {
-    
-                  _id: ObjectId(productDetails.cart),
-                
-                },
-                {
-                
-                    $pull: { products:{item:ObjectId(productDetails.product)} }, // Remove the product from user Cart
-                
-                }
                 ).then((data)=>{
     
-                    // console.log(data);
-    
                     resolve({cartProductRemoved:true});
-                    // Send a status to Ajax call as boolean inside aobject, for indicating the product removal 
+                    // Send a status to Ajax call as boolean inside aobject, for indicating the product removal
```

Comment:
Change it as follows:
// Send a response with a status to Ajax call as a boolean inside an object, for indicating the product removal